torchtitan/utils.py

```diff
@@ -36,6 +37,24 @@ def _warn_overwrite_env(env, val):
     os.environ[env] = val
 
 
+def set_determinism(job_config: JobConfig) -> None:
```

Comment:
just a `random_seed` as input arg, of type `Optional[int]`

Reply:
updated, though just used 'seed' because putting random with it when it's used for determinism makes it potentially confusing imo.